docs/userguide/networking/work-with-networks.md

```diff
@@ -57,7 +57,7 @@ $ docker network inspect simple-network
             "Config": [
                 {
                     "Subnet": "172.22.0.0/16",
-                    "Gateway": "172.22.0.1/16"
+                    "Gateway": "172.22.0.1"
```

Comment:
Looks like you missed some in this document; there's a couple of other examples as well